en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+Optionally, you can configure the `apiUsageOvertimeFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">apiUsageOvertimeFilter</td>
+<td valign="top"><a href="#apiusageovertimefilter">APIUsageOverTimeFilter</a></td>
+<td>
+
+API usage overtime filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getAPIUsageByBackendOverTime</strong></td>
+<td valign="top"><a href="#apiusagebybackendovertime">APIUsageByBackendOverTime</a></td>
+<td>
+
+Returns API usage by each backend over time for each time interval (calculates according to the granularity value)
+within the specified time range for the given combination of tenant ID, environment ID, and organization ID.
+Optionally, you can configure the `apiUsageByBackendOverTimeFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">apiUsageByBackendOverTimeFilter</td>
+<td valign="top"><a href="#apiusagebybackendovertimefilter">APIUsageByBackendOverTimeFilter</a></td>
+<td>
+
+API usage by backend overtime filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getResourceUsage</strong></td>
+<td valign="top"><a href="#resourceusages">ResourceUsages</a></td>
+<td>
+
+Returns API usage by resource within the specified time range for the given combination of tenant ID, environment
+ID, and organization ID. Optionally, you can configure the `resourceUsageFilter` parameter to further filter the
+results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">resourceUsageFilter</td>
+<td valign="top"><a href="#resourceusagefilter">ResourceUsageFilter</a></td>
+<td>
+
+Resource usage filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getTotalTrafficByAPI</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Returns the total hit count for an API across all the data available time range for the given combination of tenant
```

Comment:
"across all the data available time range" is unclear. Rephrase it to make it clearer.